contributions/presentation/week3/gchang-wska/README.md

```diff
@@ -0,0 +1,20 @@
+# Presentation Proposal
+
+## Members
+Gabriel Chang (gchang@kth.se)
+Github username: ChangGabriel
+
+William Skagerström (wska@kth.se)
+Github username: wska
+
+## Topic
+An introduction to code coverage
```

Comment:
Code coverage is a broad topic and an introduction to code coverage could be a whole course. Instead, you can focus on a more narrow topic, such as "code coverage in continuous integration".